addons/account/tests/common.py

```diff
@@ -285,6 +285,12 @@ def setup_multi_currency_data(cls, default_values=None, rate2016=3.0, rate2017=2
             'currency_subunit_label': 'Silver',
             **default_values,
         })
+        rate0 = cls.env['res.currency.rate'].create({
```

Comment:
Wouldn't it be faster to do a single create with the 3 values directly?

I know that later, we directly added them to the `foreign_currency` creation but I don't remember if it changed things in some tests (if tests added their own rates), so I won't force that.

Reply:
It's very unlikely to change anything since it's a very simple model without any computes/etc.
It's basically just one super quick `INSERT INTO`.

Did the changes anyway